modules/claims_api/config/initializers/okcomputer.rb

```diff
@@ -96,11 +123,22 @@ def name
 OkComputer::Registry.register 'mpi', MpiCheck.new
 OkComputer::Registry.register 'bgs-vet_record', BgsCheck.new('vet_record')
 OkComputer::Registry.register 'bgs-corporate_update', BgsCheck.new('corporate_update')
-OkComputer::Registry.register 'bgs-intent_to_file', BgsCheck.new('intent_to_file')
-OkComputer::Registry.register 'bgs-claimant', BgsCheck.new('claimant')
 OkComputer::Registry.register 'bgs-contention', BgsCheck.new('contention')
-OkComputer::Registry.register 'bgs-ebenefits_benefit_claims_status', BgsCheck.new('ebenefits_benefit_claims_status')
-OkComputer::Registry.register 'bgs-tracked_items', BgsCheck.new('tracked_items')
+
+OkComputer::Registry.register 'localbgs-find_poa_by_participant_id',
+                              FaradayBGSCheck.new('find_poa_by_participant_id')
+OkComputer::Registry.register 'localbgs-find_by_ssn', FaradayBGSCheck.new('find_by_ssn')
+OkComputer::Registry.register 'localbgs-find_poa_history_by_ptcpnt_id',
+                              FaradayBGSCheck.new('find_poa_history_by_ptcpnt_id')
+OkComputer::Registry.register 'localbgs-find_benefit_claims_status_by_ptcpnt_id',
+                              FaradayBGSCheck.new('find_benefit_claims_status_by_ptcpnt_id', '600033698')
+OkComputer::Registry.register 'localbgs-find_benefit_claim_details_by_benefit_claim_id',
+                              FaradayBGSCheck.new('find_benefit_claim_details_by_benefit_claim_id', '600382810')
+OkComputer::Registry.register 'localbgs-insert_intent_to_file', FaradayBGSCheck.new('insert_intent_to_file', {})
+OkComputer::Registry.register 'localbgs-find_tracked_items', FaradayBGSCheck.new('find_tracked_items', '600382810')
+OkComputer::Registry.register 'localbgs-find_intent_to_file_by_ptcpnt_id_itf_type_cd',
+                              FaradayBGSCheck.new('find_intent_to_file_by_ptcpnt_id_itf_type_cd')
+
```

Comment:
we'll want to add the general service WSDL endpoint to the local_bgs service, or create an inherited class that we can use for this. If we make these requests specific to the above methods we'll fail in production because these test users (participant IDs) don't exist. 

Similar to v1, we want something like `local-bgs_claimant` if the request uses the Claimant Service. then we'll just make a GET to the WSDL and confirm we see a 200.

Reply:
Refactored to add a healthcheck method to local_bgs; updated OKComputer to use healthcheck method and register services.